LICENSE

```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
+
+Copyright (c) 2007 - present, Python-Markdown
```

Comment:
I am not comfortable with removing the other copyright notices here. Either they all need to be left in or we are not making any change.

```
Copyright 2004, 2005, 2006 Yuri Takhteyev (v. 0.2-1.6b)
Copyright 2004 Manfred Stienstra (the original version)
```

I suppose with express permission from the named individuals, we could remove them. But not without. Unfortunately, I have no way of contacting Manfred and Yuri hasn't responded to my attempts to contact him for some years now. We may be stuck with this license as-is.

Reply:
I added the copyright information from the LICENSE.md file on master ✅.